A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/MFD/A320_Neo_MFD.html

```diff
@@ -81,7 +81,8 @@
                         <svg>
                             <g id="Waypoint_Group">
                                 <text id="WP_Name" class="Large" x="72%" y="0%" alignment-baseline="baseline"></text>
-                                <text id="WP_Track_Value" class="Value" x="100%" y="0%" alignment-baseline="baseline">---&deg</text>
+                                <text id="WP_Track_Value" class="Value" x="93.5%" y="0%" alignment-baseline="baseline">---&deg</text>
```

Comment:
Should this just be `---` now rather than `---°`?

Reply:
@tracernz if you mean the case when there is no waypoint loaded, the degree symbol would still be visible since it is added as an on-screen character. Earlier it was hard-coded (if that's the right term :P) in the form of " ---° " if no waypoint distance to be displayed, and " +String.fromCharCode(176) " if waypoint distance is to be displayed. I have basically tried to make the degree symbol independent of the value/no value situations

Reply:
Yes, looks good BTW. I mean this line I've commented on though, shouldn't you replace `---&deg` with `---`, as you've now got a separate degree symbol on the following line. I think from very quickly skimming the code it would be shown in this initial state when no waypoints have been loaded...

Reply:
> Yes, looks good BTW. I mean this specific line I've commented on though, shouldn't you replace `---&deg` with `---`, as you've now got a separate degree symbol on the following line. I think from very quickly skimming the code it would be shown in this initial state when no waypoints have been loaded...


Makes sense. Thanks for the feedback, it really helps newbies like me. Will try implementing it and see what it actually does :)